src/chi/components/expansion-panel/expansion-panel.scss

```diff
@@ -231,9 +243,9 @@ $transition-duration: 0.2s;
     .chi-epanel__body {
       background-color: white;
       border: 0.0625rem solid $expansion-panel-border-color;
+      border-radius: $expansion-panel-body-border-radius;
       box-shadow: map-get($shadow, 1);
-      margin-bottom: 0.5rem;
-      margin-top: 1.25rem;
+      margin: $expansion-panel-body-margin;
```

Comment:
I would consider removing the margin-left from `chi-epanel__body` when `-no-step` is present on `chi-epanel` to ensure `chi-epanel__body` aligns consistently with `chi-epanel__title` in all scenarios.